src/util/simple_table.hpp

```diff
@@ -0,0 +1,44 @@
+#pragma once
```

Comment:
Is this needed? Can't the user use a `map` or `unordered_map`? This is used in one location, so I don't know if it is a good idea to make extra library code to support it.

Reply:
It's really low overhead and also very simple. To be honest, I find myself wanting to use something like this very often in code generally, and there may well be uses of `map` in `arbor` that are only there because a simple table lookup did not already exist.